pkgs/tools/misc/libbitcoin/libbitcoin.nix

```diff
@@ -34,8 +34,6 @@ in stdenv.mkDerivation {
     platforms = platforms.linux ++ platforms.darwin;
     maintainers = with maintainers; [ chris-martin ];
 
-    # https://wiki.unsystem.net/en/index.php/Libbitcoin/License
-    # AGPL with an additional clause
     license = licenses.agpl3;
```

Comment:
The link is obsolete but the comment is not: the project claims to be licensed under "AGPL with a lesser clause" (where "lesser clause" must be the additional clause of LGPL: https://en.wikipedia.org/wiki/GNU_Lesser_General_Public_License#Differences_from_the_GPL)

Reply:
I don't really understand. Is it AGPL or LGPL? I also couldn't find any difference between [their license](https://libbitcoin.org/files/license.txt) and the [AGPL](https://www.gnu.org/licenses/agpl-3.0.en.html), although admittedly I didn't look very hard.

Reply:
Meaning, you can link it dynamically without disclosing your source code or licensing under the same license, but if you link it in statically, you should use the same license (AGPL with lesser clause).